### PR TITLE
Milestone 3

### PR DESCRIPTION
**Pull request by AbigiyaTY**
# Update contact form

In this milestone we:

- Updated the contact form section in the index.html
- Add styling to the contact form using CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -189,18 +189,15 @@ <h2>CONTACT US</h2>
...
-                </div>
+            <form action="https://formspree.io/f/xbjwvdqb" method="post">
+                <input type="text" id="name" name="name" maxlength="30" placeholder="Text input" required />
+                <br />
+                <br />
+                <input type="email" id="email" name="email" placeholder="jeremy@gmail.com" required /><br />
+                <br />
+                <textarea name="comments" id="comments" placeholder="Enter text here" maxlength="500" rows="8" cols="40" required></textarea>
+                <button class="button2" type="button">See Project</button>
             </form>
```
Comment (danmainah) [inline index.html]: - [ ] Kindly ensure your form works, currently, I cant submit data.

File: css/style.css [hunk 2]
```diff
@@ -358,6 +337,27 @@ form .textarea textarea {
...
+}
+
+form input {
+  height: 48px;
+  width: 283px;
+}
+
+form {
+  margin-left: 12px;
+}
```
Comment (danmainah) [inline css/style.css:348]: - [ ] Kindly ensure the elements in your form are aligned correctly, also the styling of this section should match the Figma design given, below is a comparison of the designs.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @AbigiyaTY , @lorenzoworx and @AlierPM , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by victorokoroji — APPROVED** [review]
Hi @AbigiyaTY  

# 🟢  Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The commit messages are meaningful ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**